server/odc-service/src/main/resources/builtin-snippet/mysql-snippet.yml

```diff
@@ -84,7 +84,7 @@
   body: |
     SELECT
       `user`,
-      `account_locked`,
```

Comment:
account_locked invovled from mysql 5.7, however ob 3.x has no account_locked column.
here use password_expired instead, which is involved from mysql 5.6.